lib/t_web/views/match_view.ex

```diff
@@ -38,6 +38,74 @@ defmodule TWeb.MatchView do
     |> maybe_put("voicemail", voicemail)
   end
 
+  def render("interaction.json", %{interaction: interaction}) do
+    %Interaction{data: %{"type" => type}} = interaction
+    render_interaction(type, interaction)
+  end
+
+  defp render_interaction("slots_offer" = type, interaction) do
+    %Interaction{id: id, to_user_id: picker, data: %{"slots" => slots}} = interaction
+
+    %{
+      "id" => id,
+      "type" => type,
+      "slots" => slots,
+      "inserted_at" => datetime(id),
+      "picker" => picker
+    }
+  end
+
+  defp render_interaction("slot_accept" = type, interaction) do
+    %Interaction{id: id, data: %{"slot" => slot}} = interaction
+    %{"id" => id, "type" => type, "selected_slot" => slot, "accepted_at" => datetime(id)}
```

Comment:
`by_user_id` matters here for frontend

Reply:
also i would rename all timestamps for different interactions to `inserted_at`

Reply:
also I would rename all pickers / senders / callers into `by_user_id`

Reply:
> by_user_id matters here for frontend

https://github.com/getsince/test3/pull/540/commits/74c5afbb460a2c419b4a059c186349f598dfe231

Reply:
> also i would rename all timestamps for different interactions to inserted_at

https://github.com/getsince/test3/pull/540/commits/8b775fa2c7f30fc0aad1a61aa8738530e6dd4728

Reply:
> also I would rename all pickers / senders / callers into by_user_id

https://github.com/getsince/test3/pull/540/commits/0e39b08b61b130fa893092c2c40796c0fa063279